src/utils/oracle.ts

```diff
@@ -0,0 +1,36 @@
+import { getOracleInstance } from './abi';
+import BigNumber from 'bignumber.js';
+import { Token } from '../types';
+import { BigNumberish } from 'ethers';
+import { toUnits, USDC_DECIMALS } from './commonUtils';
+
+export const getTokenUnitPrice = async (token: Token): Promise<BigNumber> => {
```

Comment:
for later we will need to make this functions accept arrays

Reply:
I will add it now.

Reply:
Done!